server/sonar-server/src/main/java/org/sonar/server/es/NewIndex.java

```diff
@@ -245,24 +255,33 @@ public StringFieldBuilder termVectorWithPositionOffsets() {
     }
 
     /**
-     * "index: no" -> Don’t index this field at all. This field will not be searchable.
-     * By default field is "not_analyzed": it is searchable, but index the value exactly
+     * "index: false" -> Make this field not searchable.
+     * By default field is "true": it is searchable, but index the value exactly
      * as specified.
      */
     public StringFieldBuilder disableSearch() {
       this.disableSearch = true;
       return this;
     }
 
+    public StringFieldBuilder store() {
+      this.store = true;
+      return this;
+    }
+
     public NewIndexType build() {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor this method to reduce its Cognitive Complexity from 18 to the 15 allowed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS3776)